free-2.md

```diff
@@ -11,41 +11,118 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Mudders get a lot of email. 
+Many people subscribe to dorm chats and occasionally receive
+email with interesting content but a lot of times they get useless
+emails (these range from counting threads, to emails asking for rides from
+airports, and to ASHMC senate summary emails).
+
+As an example, these are a series of emails sent to a certain dorm's chat
+list
+
+![EDC emails](/images/emails.png)
+
+people would want emails at the top of the list and the bottom of the list.
+However, all of the emails in between sent over a few hours, would be 
+undesirable.
+
+A language for filtering emails with enough flexibility in configuration could
+cut down on these unwanted emails reaching their inboxes. The emails could
+instead be redirected towards a folder for them to check occasionally.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would allow for more customization in order to effectively
+filter out emails. 
+This customization is key since different users would want to receive and 
+filter out different types of emails.
+The customization could also be used to specifically target types of emails.
+For example, a user could specify that they do not want emails that only contain
+numbers and a sign-off. 
+This would then filter out counting threads.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I don't mind nonsensical emails from dorm chats but a lot of people do.
+I would be happy to help people eliminate annoyances in their lives.
+
+I came up with the idea when someone mentioned that another person has a really
+complex email filtration system.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Filter out unwanted email types.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would write a configuration file. This file would contain something
+like
+
+```
+name: "ISO war"
+keywords: ["ISO"]
+start-frequency: 60
+start-count: 3
+step-frequency: 3600
+to: "ISO"
+```
+
+This means, filter out emails that are considered to be "ISO war" emails. These
+emails contain the word "ISO". 
+Filter them out if more than 3 such emails are 
+sent over the course of 60 seconds and continue filtering them out until the 
+frequency drops below 1 email per 3600 seconds. 
+These filtered emails should go to a folder labeled "ISO".
+
+A configuration file using JSON or XML-like syntax would give the system a good
+level of customization, provided that the language has enough keys.
```

Comment:
This makes it feel like it isn't a DSL so much as a program with nice config files